commercelayer/commercelayer-react-components
# Update customer payment source of a subscription after order place

## What I did

Modified the `PlaceOrderReducer` introducing a new procedure to update the customer payment source of current order's subscription.

Once the order is placed by the reducer's flow, and it is confirmed that it has subscriptions, the new procedure will take care about:
- retrieving current order including `payment_source`, `order_subscription` and `order_subscription.customer_payment_source`
- checking if retrieved `payment_source` exists
- updating `order_subscription.customer_payment_source` setting its relationship to point to order's `payment_source`

## How to test

If you have an `order_subscription` with status `active` and an empty or expired / not working `customer_payment_source` you can try to checkout subscription most recent order with `Unpaid` status and place the order with an updated / valid payment source. Then looking at the `customer_payment_source` of the `order_subscription` you should see it updated.

## Checklist
